controller/api/destination/server.go

```diff
@@ -231,7 +231,13 @@ func (s *server) GetProfile(dest *pb.GetDestination, stream pb.Destination_GetPr
 				if !ok {
 					opaquePorts = s.defaultOpaquePorts
 				}
-				endpoint, err = toWeightedAddr(podSet.Addresses[podID], opaquePorts, s.enableH2Upgrade, s.identityTrustDomain, s.controllerNS, log)
+
+				skippedInboundPorts, err := getPodSkippedInboundPortsAnnotations(pod)
+				if err != nil {
+					log.Errorf("failed getting ignored inbound ports annoatation for pod: %s", err)
```

Comment:
Typo, and then match the way we word other errors in this function:
```suggestion
					log.Errorf("failed to get ignored inbound ports annotation for pod: %s", err)
```